com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/ProcessStepHelper.java

```diff
@@ -73,24 +81,35 @@ private void init(DelegateExecution context, StepPhase initialPhase) {
         this.taskIndex = Integer.toString(computeTaskIndex(context, initialPhase, isInError));
     }
 
-    private boolean isInError(DelegateExecution context) {
-        Job job = getJob(context);
-        if (job == null) {
-            return false;
+    private String getCurrentActivityId(DelegateExecution context) {
```

Comment:
This method should be below storeExceptionInProgressMessageService if it's used only there.